app/components/index/Youtube.tsx

```diff
@@ -0,0 +1,99 @@
+"use client";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import { Youtube } from "@/types/commons/store";
+import { useState } from "react";
+import Image from "next/image";
+
+const settings = {
+  dots: true,
+  infinite: true,
+  speed: 500,
+  slidesToShow: 1,
+  slidesToScroll: 1,
+  arrows: false,
+};
+
+export default function Youtube(props: Youtube) {
+  const list = props.list;
+  const [modalIsOpen, setModalIsOpen] = useState<boolean>(false);
+  const [selected, setSelected] = useState<string>("");
+  const [selectedTitle, setSelectedTitle] = useState<string>("");
+
+  return (
+    <div>
+      <Slider {...settings}>
+        {list.map((item) => (
+          <div
+            key={item.src}
+            className="w-[100%] h-[300px]"
+            onClick={() => [
+              setSelected(item.src),
+              setSelectedTitle(item.title),
+              setModalIsOpen(true),
+            ]}
```

Comment:
배열로 리턴 되어야 하는 사유가 있을가용 ?? 궁금합니당 ~~

Reply:
여러개 함수가 동시에 실행되어야 해서 배열로 만들었습니다!

Reply:
동시에 실행되어야 한다는건 비순차적 동시성이 보장되어야 한다는건가요 ?
그렇다면 Promise 를 사용하는 방법도 있는데 저 스테이트 체인지에 굳이 동시성이 보장이 되어야 할 필요는 없을꺼 같아보입니당
저는 개인적으로 
```suggestion
            onClick={() => {
              setSelected(item.src);
              setSelectedTitle(item.title);
              setModalIsOpen(true);
            }}
```
와 같은 형태는 어떤지 의견이 궁금합니다 ~

Reply:
알겠습니다!